heroic-component/src/main/java/com/spotify/heroic/metric/Metric.java

```diff
@@ -40,7 +40,7 @@ static Metric invalid() {
         return invalid;
     }
 
-    Comparator<Metric> comparator = (a, b) -> Long.compare(a.getTimestamp(), b.getTimestamp());
+    Comparator<Metric> comparator = Comparator.comparingLong(Metric::getTimestamp);
```

Comment:
What's this change about, it seems unrelated(?)

Reply:
I was looking at how metrics get ordered (needed to sort them in the IT) and updated this. Its the exact same logic just less verbose.